### PR TITLE
environment scenarios

### PR DESCRIPTION
**Pull request by JaskRendix**
Following #1342
Contain:
- Fixed time variables on player.py;
- Fixed spyder_tunnel, it was grass, now cave;
- Fixed spyder_wayfarer_inn2, there were two identical set_variable;
- ~~Fixed coordinates environments, now 0 and 0, before it was 0 and 16;~~
- Added night 32,0 and day 16,0 cycle to open spaces (where there was already set_variable environment:xxx);
- Added interior.json for interiors; used the paper island and the default bg for battles;
- Fixed an issue for the interiors with snow #1344 

TMX files:
- spyder_candy_town.tmx (day/night) ---> set_variable environment:grass
- spyder_citypark.tmx (day/night) ---> set_variable environment:grass
- spyder_cotton_town.tmx (day/night) ---> set_variable environment:grass
- spyder_cotton_tunnel.tmx (interior) ---> set_variable environment:cave
- spyder_dragonscave.tmx (interior) ---> set_variable environment:cave
- spyder_dryadsgrove.tmx (day/night) ---> set_variable environment:grass
- spyder_flower_city.tmx (day/night) ---> set_variable environment:grass
- spyder_leather_town.tmx (day/night) ---> set_variable environment:grass
- spyder_paper_town.tmx (day/night) ---> set_variable environment:grass
- spyder_route1.tmx (day/night) ---> set_variable environment:grass
- spyder_route2.tmx (day/night) ---> set_variable environment:grass
- spyder_route3.tmx (day/night) ---> set_variable environment:sand
- spyder_route4.tmx (day/night) ---> set_variable environment:grass
- spyder_route5.tmx (day/night) ---> set_variable environment:grass
- spyder_route6.tmx (day/night) ---> set_variable environment:grass
- spyder_routea.tmx (day/night) ---> set_variable environment:grass
- spyder_routec.tmx (day/night) ---> set_variable environment:ocean
- spyder_timber_town.tmx (day/night) ---> set_variable environment:grass
- spyder_tunnel_below.tmx (interior) ---> set_variable environment:cave
- spyder_tunnel.tmx (interior) ---> set_variable environment:cave

Interior:
- spyder_dojo1.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_dojo2.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_dojo3.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_mansion_basement.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_mansion_top.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_mansion.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_nimrod_bottom.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_nimrod_middle.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_nimrod_top.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_omnichannel1.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_omnichannel2.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_omnichannel3.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_omnichannel4.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_radiotower.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_scoop1.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_scoop2.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_scoop3.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_scoop4.tmx (interior) ---> set_variable environment:snow -> now "interior"
- spyder_wayfarer_inn1.tmx (interior) ---> missing -> now "interior"
- spyder_wayfarer_inn2.tmx (interior) ---> set_variable environment:snow -> now "interior"

Paper
paper
Bg
battle_bg03

### CODE REVIEW

File: mods/tuxemon/maps/spyder_leather_town.tmx [hunk 1]
```diff
@@ -396,13 +396,19 @@
     <property name="cond2" value="is player_facing up"/>
    </properties>
   </object>
-  <object id="301" name="Environment" type="event" x="16" y="0" width="16" height="16">
+  <object id="301" name="Environment Day" type="event" x="0" y="0" width="16" height="16">
```
Comment (kerizane) [inline mods/tuxemon/maps/spyder_leather_town.tmx]: Doesn't multiple objects on the same square make it difficult to read?

Comment (JaskRendix) [reply mods/tuxemon/maps/spyder_leather_town.tmx]: I think you're right, I'll update it.

**Comment by JaskRendix** [comment]
I need to check a thing.

**Review by kerizane — APPROVED** [review]

**Comment by kerizane** [comment]
Good work!

**Pull request merged**